features/FEATURE_BLE/targets/TARGET_STM/TARGET_NUCLEO_WB55RG/hw.h

```diff
@@ -0,0 +1,310 @@
+/**
+  ******************************************************************************
+  * @file    hw.h
+  * @author  MCD Application Team
+  * @brief   Hardware
+  ******************************************************************************
+  * @attention
+ *
+ * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics.
+ * All rights reserved.</center></h2>
+ *
+ * This software component is licensed by ST under Ultimate Liberty license
```

Comment:
This shall be  licensed under permissive license as stated in our license guide https://os.mbed.com/docs/mbed-os/v5.11/contributing/index.html#licensing

The drivers are "This software component is licensed by ST under BSD 3-Clause license" . It would be good to have this as well.

Reply:
@0xc0170 WIll check this and update ! thanks

Reply:
@0xc0170  Ok. We've now agreed for updating the licence headers - which is now done in the PR.